pyanaconda/ui/gui/spokes/password.py

```diff
@@ -27,6 +27,7 @@
 from pyanaconda.ui.gui.utils import set_password_visibility
```

Comment:
The commit message seems to be unrelated to the actual changes -- more details needed.